gapis/server/server.go

```diff
@@ -298,6 +299,20 @@ func (s *server) DCECapture(ctx context.Context, p *path.Capture, requested []*p
 	return trimmed, nil
 }
 
+
+func (s *server) GetGraphVisualizationFile(ctx context.Context, p *path.Capture) (string, error) {
+	ctx = log.Enter(ctx, "Inside GetGraphVisualizationFile function")
```

Comment:
Can you make this match the other RPC calls in this file:
```
ctx = status.Start(ctx, "RPC GetGraphVisualizationFile")
defer status.Finish(ctx)
ctx = log.Enter(ctx, "GetGraphVisualizationFile")
```

Reply:
Done!!

Reply:
I have push a new commit in response to changes requested . Please review